doc/installation.rst

```diff
@@ -3,6 +3,18 @@
 Installation
 ============
 
+Before compiling GIMIC you need to make sure that you have installed
+all of the following packages, see also the ``requirement.txt`` file.
+You need at least cython and runtest. ::
+
+* cython
+* numpy
+* pyparsing
+* pyyaml
+* sphinx
+* sphinx_rtd_theme
```

Comment:
pyparsing, pyyaml, sphinx, and sphinx_rtd_theme should not be listed here. I will at some point switch to pipenv where it is clearer which packages are for running, which for testing, and which for developing.

Reply:
I included: You need at least cython and runtest -- For the time being this looks ok to me. We can always update the documentation another time.

Reply:
"Inexperienced" users will however try to install these with sudo. I think we should really remove these from this list.

Reply:
Maybe we should recommend to install Anaconda2 and use conda install XXX to get all the packages. Inexperienced and pragmatic users would like this.

Reply:
Just to be clear: I do not mind removing the stuff from the list. Instead I try to figure out a user friendly installation way.

Reply:
I agree.

Reply:
Would it be acceptable to recommend installation of Anaconda2 then? If yes I will update the documentation and file a new pull request.

Reply:
Yes I think this is a good idea. Better than letting users sudo install things.